src/resolver.ts

```diff
@@ -53,6 +53,7 @@ export async function resolve(
         results.projects.push(...projects);
 
         // ========= ENS Names =========
+        q = q.toLowerCase(); // Lowercasing query to conform with ENS name syntax
```

Comment:
Nit: could we instead of mutating the query variable just create a new one, and then use that?

```suggestion
     normalizedQuery = q.toLowerCase();
```

Reply:
Yeah sure, we can avoid mutating the state, makes sense. 👍